rpcs3/Emu/Audio/XAudio2/XAudio2Thread.cpp

```diff
@@ -11,11 +11,14 @@ XAudio2Thread::~XAudio2Thread()
 	if (m_source_voice) Quit();
 }
 
+XAudio2Thread::XAudio2Thread() : m_xaudio2_instance(nullptr), m_master_voice(nullptr), m_source_voice(nullptr)
```

Comment:
Only m_xaudio2_instance could be CComPtr so I don't see much reason to use them.